### DISCUSSION TITLE
Enabling Clipboard, and other key combos

### DISCUSSION THREAD

**Discussion by tcgm** [Voltstro-Studios/UnityWebBrowser#316] [Q&A]
How do I forward clipboard copy and paste to the UWB browser? It's not happening automatically. I know Unity is getting the copy and paste data because it pastes into text fields just fine, but the browser seems immune to it.

**Comment by Voltstro** [comment]
Fix for #335 has also fixed keyboard shortcuts like CTR+C or CTR+A.
